openassessment/fileupload/backends/__init__.py

```diff
@@ -7,7 +7,10 @@
 
 
 def get_backend():
-    # Use S3 backend by default (current behaviour)
+    # .. setting_name: ORA2_FILEUPLOAD_BACKEND
+    # .. setting_default: s3
```

Comment:
@morenol: After merging, I realized that the default should probably include quotes like `'s3'`.  Thoughts?  Would you mind updating?

Reply:
Yes. I'll create a new PR.